test/functional-tests/stepDefinitions/searchSteps.js

```diff
@@ -0,0 +1,99 @@
+
+let SearchPage = require('../pageObjects/searchPage.js');
+let TestData = require('../utils/TestData.js');
+let chai = require("chai").use(require("chai-as-promised"));
+let baseSteps = require('./baseSteps.js');
+let expect = chai.expect;
+
+var { defineSupportCode } = require("cucumber");
+
+defineSupportCode(function ({ Given, When, Then, Before, After }) {
+
+  Given(/^I am on the search page$/, async function () {
+    searchPage = new SearchPage();
```

Comment:
should we initialise the SearchPage once outside the steps at the class level then use the same instance in all the steps

Reply:
Sure and done.